discojs/discojs-core/src/async_informant.ts

```diff
@@ -1,4 +1,4 @@
-import { AggregatorBase } from './aggregator'
+import { type AggregatorBase } from './aggregator'
```

Comment:
I see you're adding types to imports everything, what changed?

Reply:
eslint asked me to and I happily obliged :bow:
it's [a new syntax](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export) to help bundlers by avoiding any reference to types that are not instanciated